examples/tests/invalid/interface.test.w

```diff
@@ -42,3 +44,26 @@ interface IMissingParamTypes {
   method1(a, b): void;
   //^ Expected type annotation
 }
+
+// Can't implement preflight interface on inflight class
+interface IPreflight {
+  method1(): void;
+}
+inflight class CImplPreflightIface impl IPreflight {
+  pub method1(): void {}
+}
+
+// Can't extend preflight interface on inflight interface
+inflight interface IInflightExtendsPreflight extends IPreflight {
+  method2(): void;
+}
+
+// Inflight interfaces can't extend JSII interfaces
+inflight interface IInflightExtendsJsii extends jsii_fixture.ISomeInterface {
+  method(): void;
+}
+
+// Inflight classes can't implement JSII interfaces
+inflight class CInflightImplJsii impl jsii_fixture.ISomeInterface {
+  pub method(): void {}
+}
```

Comment:
Would this also be an error?
```js
inflight interface IInflightIface {
  pub method(): void {}
}

inflight interface IInflightExtendsResource impl IInflightIface, std.IResource {
  pub method2(): void {}
}
```

Reply:
Yes, `std.IResource` is a JSII imported interface and therefore it's a **preflight**  interface. So you get this error:
```
inflight interface IInflightExtendsResource extends IInflightIface, std.IResource {
                   ^^^^^^^^^^^^^^^^^^^^^^^^ Inflight interface IInflightExtendsResource cannot extend a preflight interface IResource
```